.github/workflows/test-report.yml

```diff
@@ -0,0 +1,16 @@
+name: 'Test Report'
```

Comment:
This one didn't run, is the one that should make sense of the test results and present them on a beautiful way on the PR. 

I will explore why after merging (because perhaps it needs to be on `develop` in order to run....